cfgov/v1/migrations/0240_alter_cfgovcontentowner_slug.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.18 on 2023-05-15 16:04
```

Comment:
Did you make another change not included in this PR? Is there a change here that's impacting these field on the base model?

Reply:
I did not... not sure what that's about.